pkg/test-infra/tidb/recommend.go

```diff
@@ -212,6 +213,22 @@ func RecommendedTiDBCluster(ns, name string, clusterConfig fixture.TiDBClusterCo
 						},
 					},
 				},
+				TiCDC: &v1alpha1.TiCDCSpec{
+					Replicas: int32(clusterConfig.TiCDCReplicas),
+					ComponentSpec: v1alpha1.ComponentSpec{
+						Image: util.BuildImage("ticdc", clusterConfig.ImageVersion, clusterConfig.TiCDCImage),
+					},
+					ResourceRequirements: corev1.ResourceRequirements{
+						Requests: corev1.ResourceList{
+							fixture.CPU:    resource.MustParse("1000m"),
+							fixture.Memory: resource.MustParse("1Gi"),
+						},
+						Limits: corev1.ResourceList{
+							fixture.CPU:    resource.MustParse("1000m"),
```

Comment:
Is 1000m enough? (I'm not sure.)

Reply:
According to @5kbpers said, TiCDC occupies a lot of CPU for sorting, but since the TPS of our workloads usually is moderate, I think 1 core may ok now.

A related issue about the ResourceRequirements setting #376 also concerned about the recommend mechanism, I plan to refactor this part in the future(to enable coordination with NaglfarCloud)